src/shared_modules/utils/windowsHelper.h

```diff
@@ -175,6 +192,52 @@ namespace Utils
         return ret;
     }
 
+    static std::string normalizeTimestamp(const std::string& timestamp1, const std::string& timestamp2)
+    {
+        std::string normalizedTimestamp;
+
+        if (timestamp1.size() == INSTALLDATE_REGISTRY_VALUE_SIZE)
+        {
+            if (isNumber(timestamp1))
+            {
+                size_t pos = timestamp2.find(' ');
+
+                if (pos != std::string::npos)
+                {
+                    std::string date2_trimmed = timestamp2.substr(0, pos);
+                    std::string time2_trimmed = timestamp2.substr(pos + 1);
+
+                    Utils::replaceAll(date2_trimmed, "/", "");
+                    Utils::replaceAll(time2_trimmed, ":", "");
```

Comment:
if you use the split string helper and access by index ?  for better readability.

Reply:
I don't see how using that helper would improve readability and it will also add many steps in between to achieve the same result, but I'll take a deeper look.